### PR TITLE
Fix reset password completion not generating an assertion.

### PR DESCRIPTION
**Pull request by shane-tomlinson**
issue #1508

### CODE REVIEW

File: resources/static/dialog/resources/state.js [hunk 1]
```diff
@@ -241,9 +241,14 @@ BrowserID.State = (function() {
...
-      // reset password says the password has been reset, now waiting for
-      // confirmation.
       info = info || {};
+      // reset password says the password user has confirmed that they want to
+      // reset their password.  doResetPassword will attempt to call the
+      // call the create user wsapi call.  If the wsapi call is successful,
+      // the user will be directed to the "go verify your account" message.
+
+      // We have to save off the staged email address here for when the user
+      // verifies their account and user_confirmed is called.
```
Comment (lloyd) [inline resources/static/dialog/resources/state.js:251]: this comment has grammar problems

Comment (shane-tomlinson) [inline resources/static/dialog/resources/state.js:251]: @lloyd - _I_ have grammar problems.  I'll update the comment.

**Comment by lloyd** [comment]
what am I missing here?  I see zero change in terms of code that will run in normal usage of the dialog.  Is this intended to be a comment + test _only_ change?

**Comment by lloyd** [comment]
comfirmed with @shane-tomlinson that this PR and commits merged when 2012.04.11 was merged to dev fully address the issue.  this PR is zero code.  r+

**Pull request merged**